.travis.yml

```diff
@@ -1,8 +1,9 @@
 language: go
 go:
-    - "1.14"
-    - "1.12"
```

Comment:
Testify no longer supports Go 1.12. Think it's time to move on, or is it important to keep Go 1.12 compatibility?

Reply:
Move on I think